models/public/pspnet-pytorch/model.yml

```diff
@@ -0,0 +1,160 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  pspnet-pytorch is a semantic segmentation model, pretrained on Pascal VOC <http://host.robots.ox.ac.uk/pascal/VOC/>
+  dataset for 21 object classes. The model was built on ResNetV1-50 <https://arxiv.org/pdf/1812.01187.pdf>
+  backbone and PSP segmentation head. This model is used for pixel-level prediction
+  tasks. For details see repository <https://github.com/open-mmlab/mmsegmentation/tree/master>,
+  paper <https://arxiv.org/abs/1612.01105>.
+task_type: semantic_segmentation
+files:
+  - name: pspnet_r50-d8_512x512_20k_voc12aug_20200617_101958-ed5dfbd9.pth
+    size: 196212259
+    sha256: ed5dfbd939dd2e4f78a0c2e6726a57150f3aec95e07f2cf64aeffea68a712c5b
+    source: https://download.openmmlab.com/mmsegmentation/v0.5/pspnet/pspnet_r50-d8_512x512_20k_voc12aug/pspnet_r50-d8_512x512_20k_voc12aug_20200617_101958-ed5dfbd9.pth
+  - name: models/configs/pspnet/pspnet_r50-d8_512x512_20k_voc12aug.py
+    size: 263
+    sha256: 0a36424a1dec8d068fc5676c90106a0b15914b474e36e91aad4621e26aa56aae
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/pspnet/pspnet_r50-d8_512x512_20k_voc12aug.py
+  - name: models/configs/_base_/models/pspnet_r50-d8.py
+    size: 1261
+    sha256: 8ac435c6388d9896dacde51e0f24d3d3c27f4972dad88f1eba9327341b689c25
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/_base_/models/pspnet_r50-d8.py
+  - name: models/configs/_base_/datasets/pascal_voc12_aug.py
+    size: 261
+    sha256: b61d01edccac1a2622944761a50772a54b4848cd9e4817c25551f2cccdd05a08
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/_base_/datasets/pascal_voc12_aug.py
+  - name: models/configs/_base_/datasets/pascal_voc12.py
+    size: 1930
+    sha256: b9fac8609d3c8d8a3536803b53a1222ba2d6f02af6b1638b7e78af4ab233805b
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/_base_/datasets/pascal_voc12.py
+  - name: models/configs/_base_/default_runtime.py
+    size: 321
+    sha256: 27e0d1e40cf3956e2af35b8de32436eb78f91715570ba435144b29b515b1b5a1
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/_base_/default_runtime.py
```

Comment:
mmsegmentation has wheels on PyPI; maybe it would be easier to download the last release wheel and unzip it instead of downloading the individual files? It might render some of the patches unnecessary too.

Reply:
updated mmsegmentation to the latest release, but PyPI package doesn't include config files, they are still needed to be downloaded from github